lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4373,6 +4372,13 @@ namespace Js
 /* END potentially reusable code */
 #endif /* IR_VIEWER */
 
+#if ENABLE_TTD
+    void FunctionBody::GetSourceLineFromStartOffset_TTD(const uint startOffset, ULONG* line, LONG* col)
```

Comment:
Why a different function when it is not doing anything extra and just calling GetLineCharOffsetFromStartChar?